libs/ui-lib-tests/cypress/views/forms/OperatorsForm.ts

```diff
@@ -0,0 +1,36 @@
+export default class OperatorsForm {
+  static get body() {
+    return cy.get('.pf-c-wizard__main-body');
```

Comment:
please add a `data-testid` and use cy.findByTestId

Reply:
not works. Maybe I can change in a followup